person.rb

```diff
@@ -7,11 +7,16 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   attr_reader :id # getter
   attr_accessor :name, :age # getter and setter
```

Comment:
- **[OPTIONAL]** I think we could be more consistent. We can name this `@rentals` since it contains many rentals instead of one.

- [ ] We should be able to set and get the rentals:

  ```
  attr_accessor :name, :age, :rentals
  ```